app/assets/stylesheets/activities.css.scss

```diff
@@ -74,7 +74,7 @@ article {
 
 #climbing-card{
     height: 176px;
-    background: url('climbing.jpg') center / cover;
+    background: image-url('climbing.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings